src/app/Fake.Deploy/DeploymentHelper.fs

```diff
@@ -29,7 +29,7 @@ let getActiveReleases dir = !!(dir @@ deploymentRootDir @@ "**/active/*.nupkg")
 
 /// Retrieves the NuSpec information for the active release of the given app.
 let getActiveReleaseFor dir (app : string) = 
-    !!(dir @@ deploymentRootDir @@ app @@ "/active/*.nupkg")
+    !!(dir @@ deploymentRootDir @@ app @@ "active/*.nupkg")
```

Comment:
Well, it didn't break any of the unit tests, but I certainly admit it is a big change as its used quite a lot and will possibly break scripts in the wild.  The issue originally raised is still present though.  Passing in a full path as path2 on windows just returns the full path, doing so on Linux results in a really long nested wrong path.  It is of course your decision to make.  Accepting the change would at least require some important wording in the next release notes.  On windows, it shouldn't make any difference, but it might break some things on linux that are working around the current behavior, or that expected this behavior.